pkg/utils/cluster_names_test.go

```diff
@@ -21,10 +21,19 @@ func TestUtils(t *testing.T) {
 }
 
 var _ = Describe("Utils tests", func() {
+	Context("ClusterName", testClusterName)
 	Context("ClusterNamesFromFiles", testClusterNamesFromFiles)
 	Context("ClusterNamesOrAll", testClusterNamesOrAll)
 })
 
+func testClusterName() {
+	When("sent a number", func() {
+		It("should return a cluster name", func() {
```

Comment:
I don't think we need a `When` clause here. How about:

`It("should return the correct cluster name for the given number", ...`